test/util/cmr.ts

```diff
@@ -0,0 +1,112 @@
+/* eslint-disable max-len */
+import { describe, it } from 'mocha';
+import { expect } from 'chai';
+import { getVariablesByIds } from '../../app/util/cmr';
+
+describe('util/cmr', function () {
+  describe('getVariablesByIds', function () {
+    it('returns a valid response, given a huge number of variables', async function () {
+      const ids = [
+        'V1240763852-YOCLOUD', 'V1240763854-YOCLOUD', 'V1240763844-YOCLOUD',
```

Comment:
Impressive.  Could maybe make it more concise by using a for loop to generate bogus IDs.